### PR TITLE
Fixed load multiple page with rows without rowkey

### PR DESCRIPTION
**Pull request by bhugot**
This fix an issue when loading multiple pages of rows that does have empty string rowkey.  #1554

### CODE REVIEW

File: tests/table/apis/table.entity.issues.test.ts [hunk 1]
```diff
@@ -151,7 +157,7 @@ describe("table Entity APIs test", () => {
       assert.notStrictEqual(result.etag, undefined);
     }
 
-    const maxPageSize = 250; // this should work with a page size of 1000, but fails during response serialization on my machine
+    const maxPageSize = 50; // this should work with a page size of 1000, but fails during response serialization on my machine
```
Comment (edwin-huber) [inline tests/table/apis/table.entity.issues.test.ts:160]: Hi,
could you explain the change to the page size here?  
Was it causing some problem?
Thanks.

**Review by edwin-huber — APPROVED** [review]
LGTM - Just wondering if we have an issue with paging performance / serialization... see comment.
@XiaoningLiu

File: tests/table/apis/table.entity.issues.test.ts (hunk 1 above)
Comment (bhugot) [reply tests/table/apis/table.entity.issues.test.ts:160]: Hello, It's just to match the test description 
```should return 101 entities from a paged query at 50 entities per page and single partition```

Comment (edwin-huber) [reply tests/table/apis/table.entity.issues.test.ts:160]: Thanks!

**Review by XiaoningLiu — APPROVED** [review]

**Pull request merged**